internal/metastore/model/collection.go

```diff
@@ -22,7 +22,7 @@ type Collection struct {
 	StartPositions       []*commonpb.KeyDataPair
 	CreateTime           uint64
 	ConsistencyLevel     commonpb.ConsistencyLevel
-	Aliases              []string
+	Aliases              []string          // TODO: deprecate this.
```

Comment:
Maybe it should maintain within the collection model, we need a mapping from the collection id to all aliases of itself